src/libexec/zmcbpolicydinit

```diff
@@ -74,6 +74,35 @@ initCBPolicyd() {
   else
     echo "done."
   fi
+  
+  echo -n "Adding Domains..."
+    for dom in `zmprov gad`; do sqlite3 /opt/zimbra/data/cbpolicyd/db/cbpolicyd.sqlitedb "INSERT INTO policy_group_members (PolicyGroupID,Member) VALUES ('2','@$dom');"; done
+  if [ $? != 0 ]; then
+    echo "failed."
+    exit 1
+  else
+    echo "done."
+  fi
+
+ echo -n "Adding subnets from Mynetworks global..."
+  globalSubnet=$(zmprov gacf | grep zimbraMtaMyNetworks | sed 's/127.0.0.0\/8 //g' | awk '{ print $NF }')
```

Comment:
nit: personal opinion, but do as you like: I prefer to avoid the grep, but it does mean you have to ignore a little more zmprov output noise
this:

`zmprov gacf | grep zimbraMtaMyNetworks | sed 's/127.0.0.0\/8 //g' | awk '{ print $NF }'`

becomes:

`zmprov gacf zimbraMtaMyNetworks | sed -E '/^(#|$)/d; s,127.0.0.0/8 ,,g' | awk '{ print $NF }'`

Reply:
This will actually break/not work if the entry 127.0.0.0/8 is the last entry in the list. This might work instead:
```bash
zmprov gacf zimbraMtaMyNetworks | awk -v k=zimbraMtaMyNetworks '$1 == k":" { print $0 }' | tr ' ' '\n' | tail -n +2 | grep -v -F 127.0.0.0/8 | tr '\n' ' '
```
But what about IPv6? (I might miss some context here.)